_includes/banner.html

```diff
@@ -19,30 +29,65 @@
         </button>
       </div>
     </header>
-    <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+    <div
+      class="usa-banner__content usa-accordion__content"
+      id="gov-banner"
+    >
```

Comment:
Using `gov-banner` instead of `gov-banner-default` to prevent duplicate `id`'s on the banner component guidance pages